oneview/resource_hypervisor_cluster_profile.go

```diff
@@ -452,11 +498,13 @@ func resourceHypervisorClusterProfile() *schema.Resource {
 			"status": {
 				Type:     schema.TypeString,
 				Optional: true,
+				Computed: true,
 			},
 
 			"type": {
 				Type:     schema.TypeString,
 				Optional: true,
+				Computed: true,
```

Comment:
i think it should be set as computed: true no optional

Reply:
This is read only. I will change it.